src/Protobuf/Types/Int64.elm

```diff
@@ -0,0 +1,21 @@
+module Protobuf.Types.Int64 exposing (Int32s, Int64, fromInt32s, toInt32s)
+
+import Internal.Int64
+
+
+type alias Int64 =
+    Internal.Int64.Int64
+
+
+type alias Int32s =
+    Internal.Int64.Int32s
+
+
+fromInt32s : Int32s -> Int64
```

Comment:
I saw this naming convention in the library as well. In the context of Protobuf I find `int32s` rather confusing as it could mean "signed" rather than "multiple". Maybe just `Ints`?

Reply:
Please construct using 2 `Int`s. Maybe just `fromInts`?